templates/community-viz/scripts/dscc-scripts.js

```diff
@@ -17,37 +17,62 @@
  * limitations under the License.
  */
 
-const args = process.argv.slice(2);
+const argparse = require('argparse');
 const shell = require('shelljs');
 const chalk = require('chalk');
-const script = args[0];
 
 const msg = require('./bin/message.js');
 const push = require('./bin/push.js');
 const build = require('./bin/build.js');
 
-switch (script) {
+const parser = new argparse.ArgumentParser({
+  version: '1.0',
+  addHelp: true,
+  description: 'Scripts to manage viz dev and deployment',
+});
+
+parser.addArgument(['-s', '--script'], {
+  choices: ['start', 'build', 'push', 'updateMessage'],
+  dest: 'script',
+  help: 'The script to run.',
+  required: true,
+});
+
+parser.addArgument(['-p', '--prod'], {
```

Comment:
argparse has a built in for doing booleans. You'll want an action of "storeTrue". I think I have an example of this in the code for dscc-gen.